src/admin/admin.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common';
+import { GuestService } from '../guest/guest.service';
+import { JhsService } from '../jhs/jhs.service';
+import { ObService } from '../ob/ob.service';
+import { SponsorService } from '../sponsor/sponsor.service';
+import { StudentService } from '../student/student.service';
+import { AdminController } from './admin.controller';
+import { AdminService } from './admin.service';
+
+@Module({
+   providers: [AdminService, StudentService, GuestService, JhsService, ObService, SponsorService],
```

Comment:
DIの仕方がおかしい気がする。今までのmoduleは見ていないからわからない確認したら下のツリーに報告する

```suggestion
   imports: [StudentsModule, GuestModule, JhsModule, ObModule, SponsorModule]
   providers: [AdminService],
```